### ISSUE TITLE
Refresh Ouachita Hills Theme

### ISSUE THREAD

**Issue by narthur** [OuachitaHillsMinistries/oh-theme#27]
# Resources
## AJAX
- https://codex.wordpress.org/Plugin_API/Action_Reference/wp_ajax_%28action%29 - GOOD
- https://www.smashingmagazine.com/2011/10/how-to-use-ajax-in-wordpress/ - GOOD
- http://codex.wordpress.org/AJAX
- https://wordpress.org/support/topic/wordpress-and-jquery-how-get-a-post-via-ajax
- http://codex.wordpress.org/AJAX_in_Plugins
- http://www.makeuseof.com/tag/tutorial-ajax-wordpress/
## JavaScript Dependency Management
- http://requirejs.org/ - probably better
- http://browserify.org/
## Adaptive Image Loading
- http://adaptive-images.com/
- https://github.com/MattWilcox/Adaptive-Images/issues/22#issuecomment-19578982

**Comment by narthur** [comment]
@ErinOHMin you can link to examples in this discussion if you'd like.

**Comment by ErinOHMin** [comment]
http://www.reuters.tv/
http://weimaracademy.org/

**Comment by ErinOHMin** [comment]
http://www.socialprintstudio.com/

**Comment by narthur** [comment]
@ErinOHMin What do you think of the visual direction these sites have?:
- http://nanoc.ws/ (my favorite)
- https://www.lookout.com/
- http://froont.com/
- http://hirondelleusa.org/
- http://www.philiphousenyc.com/

I love the Weimar Academy's website layout and general approach, but it's a wee bit too hipster for my personal tastes. I'm afraid it may date itself quickly. What do you think about aiming for "crisp" and "modernist" as opposed to "retro" and "hipster?"

**Comment by ErinOHMin** [comment]
I agree with your analysis of the Weimar site. The Nanoc site is very navigable, which is what I desperately want for our own website!

**Comment by narthur** [comment]
@ErinOHMin Ok, sounds good. Do we want to go with the current color scheme? If not, can you get me the new colors?

Here are the old hex color codes:

**Green**
5AB51F
497F19
B1DE77
AFCF79
57861F
6AAC24

**Blue**
094666
508BA8
ACBDCC
ABBDC5
4C8A9D
063E5E

**Comment by ErinOHMin** [comment]
I'll think about the colors.

More websites I like...
https://www.fordfoundation.org/
http://www.theaveryfoundation.ca/

**Comment by ErinOHMin** [comment]
Can our font be less "round"? Just wondering. I guess I prefer a "boxier" font, but that's just one opinion.

**Comment by narthur** [comment]
If you're referring to the logo, that's a branding issue that's a bit out of my authority, I think. If you're referring to the web font, I'm definitely open to exploring other options. A good place to find freely-usable and high-quality fonts is [Font Squirrel](http://www.fontsquirrel.com/).

**Comment by ErinOHMin** [comment]
I was thinking about the web font. I'm suspecting any concerns I have regarding the logo would not be easily addressed.

**Comment by narthur** [comment]
I believe it's best practice to create clear differentiations between fonts used; so if you can find a font that works for body text and is less similar to the logo font, that would be great.

**Comment by narthur** [comment]
For later reference: We looked at [Ostrich Proper](https://www.tylerfinck.com/work/fonts/ostrich-proper/) available for sale [here](https://www.tylerfinck.com/shop/ostrich-proper-complete/).

**Comment by ErinOHMin** [comment]
Possible alternatives to Ostrich Proper?

https://www.behance.net/gallery/28749913/HK-Grotesk-Open-Source-Typeface
https://www.behance.net/gallery/25310059/Genome-Open-Source-Typeface
